clients/gitlabrepo/client.go

```diff
@@ -283,7 +283,8 @@ func CreateGitlabClient(ctx context.Context, host string) (clients.RepoClient, e
 }
 
 func CreateGitlabClientWithToken(ctx context.Context, token, host string) (clients.RepoClient, error) {
-	client, err := gitlab.NewClient(token, gitlab.WithBaseURL(host))
+	url := "https://" + host
```

Comment:
Can you remind me why this change was needed again? I know you mentioned it, but I've forgotten and having a recorded answer is good. 

Is this just to strip the scheme from the host before sending it to deps dev? If yes, should that be an implementation detail of `func (d depsDevClient) GetProjectPackageVersions` instead?

Reply:
Yes, it's because the deps.dev client needs the domain + path but not scheme. But also, this matches how the GitHub client works more closely.